tasks/travis-pr-title-lint.js

```diff
@@ -18,25 +18,53 @@ var writeCommitMessagesDocURL =
   '/CONTRIBUTING.md#writing-commit-messages';
 
 module.exports = function(grunt) {
-  function countGitMergeCommits() {
+  function findMergeBase() {
```

Comment:
This new helper uses the `git merge-base` to get the sha1 of the branching point commit (and optionally print it if the verbose mode is enabled)